esmvaltool/cmorizers/obs/utilities.py

```diff
@@ -10,7 +11,7 @@
 from cf_units import Unit
 from dask import array as da
 
-from esmvalcore._config import get_tag_value
+from esmvalcore._citation import _collect_bibtex_citation, REFERENCES_PATH
```

Comment:
It would be nicer to implement the required code here (it's only a few lines after all), because now you are importing from a module that is [not public](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables) because it starts with an `_`, i.e. it can completely change from one minor version of ESMValCore to the next without advance notice.

Reply:
Thanks. I removed the imports.